### PR TITLE
added w3c.json and metadata to WG/ED

### PR DESCRIPTION
**Pull request by himorin**
(is shortname ok with this?)

***
<a href="https://pr-preview.s3.amazonaws.com/himorin/dom-overlays/pull/31.html" title="Last updated on May 19, 2020, 6:43 PM UTC (f2137f7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/immersive-web/dom-overlays/31/92a3949...himorin:f2137f7.html" title="Last updated on May 19, 2020, 6:43 PM UTC (f2137f7)">Diff</a>

### CODE REVIEW

File: w3c.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ {
+    "group":      109735
+,   "contacts":   ["dontcallmedom"]
+,   "shortName":  "dom-overlays"
```
Comment (dontcallmedom) [inline w3c.json:4]: `shortName` is not needed, so I suggest skipping it

File: w3c.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+ {
+    "group":      109735
+,   "contacts":   ["dontcallmedom"]
```
Comment (dontcallmedom) [inline w3c.json:3]: add yourself too? :)

Comment (himorin) [reply w3c.json:3]: Thank you for pointing this.. I suppose I need to check all immersive-web WG repos on this.
Also, should update group id for bikeshed... Let me update PR(s) shortly.

**Comment by himorin** [comment]
(updated)

**Pull request merged**